### PR TITLE
create a file, add my name and something else

### LINKED ISSUE #81: Adding Berlin
## Adding Berlin to the map

Hi everyone! This issue is to track the addition of <your-city> to the map, which can be found at:
- Longitude: 13.404954
- Latitude: 52.520007

### PR DESCRIPTION
**Pull request by NaiveJoker**
##what did I do
I type down my name
@githubteacher what do you think?
This pull requests closes #81

### CODE REVIEW

File: _pins/NaiveJoker.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: NaiveJoker
```
Comment (crichID) [inline _pins/NaiveJoker.json]: @NaiveJoker you will need to add three dashes on the first and last line `---` in order for your checks to pass.

File: _pins/NaiveJoker.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+latitude:39.9289
```
Comment (crichID) [inline _pins/NaiveJoker.json]: @NaiveJoker you must have a space after each `:`.

**Comment by brianamarie** [comment]
@NaiveJoker  :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

File: _pins/NaiveJoker.json (hunk 1 above)
Comment (NaiveJoker) [inline _pins/NaiveJoker.json]: I am sorry, I can't watch that video, so I still don't know how to make changes.

**Comment by brianamarie** [comment]
@NaiveJoker I'm sorry about the video - maybe this documentation will be helpful: https://help.github.com/articles/editing-files-in-your-repository/

**Pull request merged**